projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +149,77 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
 export function visitApparelUKTicketListingPage(){
   visitPage('apparel-uk-spa/en/GBP/my-account/support-tickets', 'apparelTicketListingPage');
 }
+
+export function getTicketDetailsFromTicketListView(){
+  let ticketDetails = [];
+  const row = cy.get('cx-customer-ticketing-list').find('tbody').get('tr').eq(1);
+  row.get('td').eq(0).invoke('text').then((x) => ticketDetails.push(x));
+  row.get('td').eq(1).invoke('text').then((x) => ticketDetails.push(x));
+  row.get('td').eq(5).invoke('text').then((x) => ticketDetails.push(x));
+
+   return ticketDetails;
```

Comment:
Nice idea, some remarks:
1) Avoid using magic numbers 0 1 5 have a look at: https://github.com/SAP/spartacus/blob/91a7e38c0cea1e7e46d453bd3c4db7ce952abe26/projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts#L123

2) There is an interface hosting ticket details, please use it: 
https://github.com/SAP/spartacus/blob/91a7e38c0cea1e7e46d453bd3c4db7ce952abe26/projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts#L20

3) Add return type to the function signature